DataFormats/Detectors/GlobalTracking/src/RecoContainer.cxx

```diff
@@ -129,6 +131,7 @@ void DataRequest::requestTRDTracklets()
   addInput({"trdtracklets", o2::header::gDataOriginTRD, "TRACKLETS", 0, Lifetime::Timeframe});
   addInput({"trdctracklets", o2::header::gDataOriginTRD, "CTRACKLETS", 0, Lifetime::Timeframe});
   addInput({"trdtriggerrec", o2::header::gDataOriginTRD, "TRKTRGRD", 0, Lifetime::Timeframe});
+  requestMap["trackletTRD"] = false;
```

Comment:
So at the moment one cannot request TRD tracklets? Shouldn't that also be optional?